src/Akka.Bootstrap.Docker/ListParser.cs

```diff
@@ -122,6 +122,7 @@ private void ParseToken()
                         _state = TokenizerState.AfterComma;
                         // consume it and go on
                         Consume();
+                        ConsumeConsecutiveWhiteSpaces();
```

Comment:
Its possible to have white spaces between opening square bracket and the start of the string token value. Need to trim white spaces else the string parser will be confused.